src/layouts/EditHomepage.jsx

```diff
@@ -873,6 +873,24 @@ export default class EditHomepage extends Component {
     } = this.state;
     const { match } = this.props;
     const { siteName } = match.params;
+
+    const hasSectionErrors = _.some(errors.sections, section => {
+      // Section is an object, e.g. { hero: {} }
+      // _.keys(section) produces an array with length 1
+      // The 0th element of the array contains the sectionType
+      const sectionType = _.keys(section)[0]
+      return _.some(section[sectionType], errorMessage => errorMessage.length > 0) === true
```

Comment:
(nitpick) since `_.keys(section)` produces an array with length 1, can't this be reduced to `section[sectionType].length > 0`